modules/serverless-concurrency-limits-configure-hard.adoc

```diff
@@ -24,9 +24,9 @@ spec:
       containerConcurrency: 50
 ----
 +
-The default value is `0`, which means that there is no limit on the number of requests that are permitted to flow into one pod of the service at a time.
+The default value is `0`, which means that there is no limit on the number of requests that are permitted to flow into one replica of the service at a time.
 +
-A value greater than `0` specifies the exact number of requests that are permitted to flow into one pod of the service at a time. This example would enable a hard concurrency limit of 50 requests at a time.
+A value greater than `0` specifies the exact number of requests that are permitted to flow into one replica of the service at a time. This example would enable a hard concurrency limit of 50 requests at a time.
```

Comment:
This is a nit but isn't "at a time" redundant when we already mention "concurrency" ? Not sure